### DISCUSSION TITLE
GUI command for v2.4

### DISCUSSION THREAD

**Discussion by GiveHimNoRest** [Security-Onion-Solutions/securityonion#11067] [2.4] [answered]
I'm installing version 2.4 after using version 2.3. During the final step, I entered the command "sudo so-analyst-install" to get the GUI dashboard, but it says it's an unrecognized command. Are there new commands in version 2.4?

**Comment by dougburks** [comment, marked as answer]
`so-analyst-install` has been renamed to `so-desktop-install`. For more information, please see:
https://docs.securityonion.net/en/2.4/desktop.html
